src/include/storage/data_table.h

```diff
@@ -166,6 +166,16 @@ class DataTable {
    */
   SlotIterator end() const;
 
+  /**
+   * @warning NOT the number of tuples currently in the data table. Such a value is not well-defined unless referring
+   * to a transactional snapshot.
+   * @return number of slots allocated for this data table.
+   */
+  uint64_t NumSlots() const {
```

Comment:
Is this something that would eventually be replaced by the metrics team?

Reply:
Removed for now.